src/components/DashboardLayout.tsx

```
@@ -24,62 +23,30 @@ import {
 } from "@/components/ui/dropdown-menu";
 import { Pencil, LinkIcon, ArrowUpRight, Trash2 } from "lucide-react";
 import { useChatStore } from "@/stores/chatStore";
-import { useRouter } from "next/navigation";
-import { ChatSession } from "@/types/chat";
+import { useRouter, useSearchParams } from "next/navigation";
 
 interface BreadcrumbItem {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'BreadcrumbItem'. Consider to delete it or rename it.

'BreadcrumbItem' is defined here:


(lint/suspicious/noRedeclare)